examples/odfe-pipes-trace-app/opentelemetry-collector/otel-collector-config.yml

```diff
@@ -7,7 +7,7 @@ receivers:
 exporters:
   otlp/2:
     endpoint: opendistro-pipes:21890
```

Comment:
Change the container name to **situp** from odfe-pipes in docker compose and change the endpoint: **situp:21890**